### PR TITLE
updated blog section to fetch from main blog website

### LINKED ISSUE #7: Fetch Latest News & Updates from blog.metabrainz.org automatically
https://musicbrainz.org fetches the latest blogs automatically whereas we currently have the news hard-coded. 
We can use the Wordpress integration for this or have a look at the legacy code of https://github.com/metabrainz/musicbrainz-server

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

Now the website shows the recent 8 blogs from the `blog.metabrainz.org` website. 

This PR fixes #7

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -703,6 +703,13 @@
   "resolved" "https://registry.npmjs.org/axe-core/-/axe-core-4.3.5.tgz"
   "version" "4.3.5"
 
+"axios@^0.24.0":
```
Comment (akshaaatt) [inline yarn.lock:706]: Let's avoid using axios and use fetch directly.

File: yarn.lock [hunk 2]
```diff
@@ -1743,6 +1750,11 @@
   "resolved" "https://registry.npmjs.org/flatted/-/flatted-3.2.4.tgz"
   "version" "3.2.4"
 
+"follow-redirects@^1.14.4":
```
Comment (akshaaatt) [inline yarn.lock:1753]: Do we really need this?

File: yarn.lock (hunk 1 above)
Comment (asubedy) [reply yarn.lock:706]: Ok on it.

**Review by akshaaatt — CHANGES_REQUESTED** [review]
Looks good, just a few more tweaks and we should be good to go!

File: yarn.lock (hunk 2 above)
Comment (asubedy) [reply yarn.lock:1753]: I didn't install this, I just installed axios with `npm install axios`

Comment (akshaaatt) [reply yarn.lock:1753]: Cool, I believe removing axios should remove this as well then.

**Comment by akshaaatt** [comment]
@asubedy you can mention in the description that this PR fixes https://github.com/akshaaatt/musicbrainz-web-next/issues/7

**Comment by asubedy** [comment]
Trying to render some text like `Loading Blogs......` in the short period when the api fetches data. I tried conditional rendering but, I don't know why after the state gets updated the component is not refreshing.

**Review by akshaaatt — APPROVED** [review]
Looks good, great work!

**Pull request merged**